### PR TITLE
Landing Page

### PR DESCRIPTION
**Pull request by ark2002**
The landing page with a component page which contains the dummy components list.

### CODE REVIEW

**Comment by vaibhav18matere** [comment]
good work. One suggestion : the sub-heading is not clearly visible. you can increase it's `font-weight`

**Comment by ark2002** [comment]
> good work. One suggestion : the sub-heading is not clearly visible. you can increase it's `font-weight`

hey ! fixed the sub-heading , you can check it out now !

**Comment by vaibhav18matere** [comment]
> > good work. One suggestion : the sub-heading is not clearly visible. you can increase it's `font-weight`
> 
> hey ! fixed the sub-heading , you can check it out now !

awesome, good to go !

**Comment by horsemaker** [comment]
image
there's no space at the bottom here, could we add some space so that it looks clean?

File: components_page.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+          <div class="header__logo-container">
+            <img
+              src="/assests/zodiaclogo.png"
+              alt="AtroUI logo"
+              class="logo__img"
+            />
+            <h2 class="header__logo">Astro UI</h2>
+          </div>
+        </a>
+        <nav class="header_nav">
```
Comment (horsemaker) [inline components_page.html]: Instead of  `header_nav`, shouldn't it be `header__nav`?

File: components_page.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+          <ul>
+            <li><a href="#">Documentation</a></li>
+            <li><a href="https://github.com/ark2002/astro-ui">Github</a></li>
+          </ul>
+        </nav>
+      </header>
+      <main class="main-content">
```
Comment (horsemaker) [inline components_page.html]: You could try to maintain a consistent naming convention. Shouldn't it be `main__content`?

File: css/components_page.css [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  font-weight: 600;
+}
+.main-content {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 2rem;
+}
+.intro_text {
```
Comment (horsemaker) [inline css/components_page.css]: `intro__text` here?

File: components_page.html (hunk 2 above)
Comment (ark2002) [reply components_page.html]: > You could try to maintain a consistent naming convention. Shouldn't it be `main__content`?

hey! Did the suggested changes , you can check it out now.

Comment (horsemaker) [reply components_page.html]: Great! 👌

**Review by horsemaker — APPROVED** [review]

**Pull request merged**